src/controllers/router.js

```diff
@@ -1,5 +1,6 @@
 const router = require('express').Router();
 
-router.get('/', require('./home.js'))
+router.get('/', require('./home.js'));
+router.get('/en/places', require('./places.js').getAll);
```

Comment:
I don't really like this layout it looks a bit confusing - think it'd be better to require ./places at the top as `placesController` or something and then do `placesController.getAll` here